pennylane/transforms/classical_jacobian.py

```diff
@@ -16,9 +16,10 @@
 """
 # pylint: disable=import-outside-toplevel
 import pennylane as qml
+from pennylane import numpy as np
 
 
-def classical_jacobian(qnode):
+def classical_jacobian(qnode, argnums=None):

```

Comment:
relatively minor, _but_ `qml.jacobian` uses `argnum` (not plural), because Autograd uses `argnum`. It always annoyed me, because it sounds like it doesn't permit sequences.

I guess the developers fixed this when they created JAX, since JAX uses `argnums` plural.

Reply:
I noticed that it is different between Autograd and JAX. Maybe it would be nice to have it the same way for `qml.jacobian` and `qml.classical_jacobian`, yes...
As `qml.jacobian` does allow for sequences and `autograd.jacobian` does _not_, I think this actually is a perfect opportunity to hint at this difference in behaviour and to rename the `qml.jacobian` argument to `argnums`.